models/nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
```

Comment:
- [ ] please make the error raised more specific so the user knows what they are looking for, this might look something like: `raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"`